frontend/packages/knative-plugin/src/actions/add-event-source.tsx

```diff
@@ -29,7 +21,7 @@ export const AddEventSourceAction = (
   return {
     id: 'event-source-add',
     label: i18next.t('knative-plugin~Event Source'),
-    icon: <EventSourceIcon />,
+    icon: <SignOutAltIcon />,
```

Comment:
Same here. If possible let us reuse the `eventSourceIcon` icon from `../utils/icons.tsx` here